run-database-tests

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("working_dir", type=str,
+                    help="Working directory of tests")
+parser.add_argument("-c","--config_file", type=str,
+                    help="Test configuration file (default: testsuite.json)",
+                    default="testsuite.json")
+parser.add_argument("-d", "--database", type=str,choices=["all","mysql","pgsql","sqlite3"],
+                    help="Run tests for specific database (all, mysql, pgsql, sqlite3).  \"all\" selects all databases listed in the testsuite configuration file. Default: \"pgsql\"",
+                    default="pgsql")
+parser.add_argument("-l", "--local", action="store_true",
+                    help="Test local database configuration (<working_dir>/config) if present; otherwise, the database(s) specified by the --database parameter are tested.",
+                    default=False)
+parser.add_argument("-v", "--verbose", action="store_true",
+                    help="Print verbose debugging messages",
+                    default=False)
+args = parser.parse_args()
+
+def main():
+    working_dir = args.working_dir
+    config_file = args.config_file
+    verbose = args.verbose
+
+    linksdir = os.getcwd()
+    linksexec = os.path.join(linksdir,"links")
+
+    def debug(s):
+        if verbose:
+            print(s)
+
+    def linkscmd(config,file):
```

Comment:
Nit: `links_cmd` is more widely-accepted Python style

Reply:
Done